### PR TITLE
Add swagger docs and integration for claims letters

### PR DESCRIPTION
**Pull request by kjduensing**
## Description of change

This PR describes the creation of Swagger docs for the `index` and `show` routes of the claim letters controller. 

## Original issue(s)
department-of-veterans-affairs/va.gov-team#49562

## Things to know about this PR

The only change in this PR is the creation of the swagger docs and integration into the main swagger docs page.

### CODE REVIEW

File: app/swagger/swagger/requests/claim_letters.rb [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+            key :required, true
+            key :type, :string
+          end
+
+          response 200 do
+            key :description, 'Single Claim Letter PDF'
+            schema do
+              key :type, :file
+            end
+          end
```
Comment (kjduensing) [inline app/swagger/swagger/requests/claim_letters.rb:54]: What other responses could we include? Does anything other than a 500 occur if there's a problem with the upstream dependency?

Comment (jerekshoe) [reply app/swagger/swagger/requests/claim_letters.rb:54]: Not that I know of, the only error we've seen so far in staging is 500

Comment (samcoforma) [reply app/swagger/swagger/requests/claim_letters.rb:54]: If your curious about the errors connect_vbms throws: https://github.com/department-of-veterans-affairs/connect_vbms/blob/master/lib/vbms/errors.rb

I think we only need to worry about 500 for our purposes

**Review by samcoforma — APPROVED** [review]

**Review by samcoforma — APPROVED** [review]

File: app/swagger/swagger/requests/claim_letters.rb [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+          key :tags, %w[claim_letters]
+          key :summary, 'Download a single PDF claim letter'
+          key :operationId, 'downloadPDFClaimLetter'
+          key :produces, ['application/pdf']
+
+          parameter do
+            key :in, :path
+            key :name, :document_id
+            key :pattern, '^[0-9a-fA-F]{8}-([0-9a-fA-F]{4}-){3}[0-9a-fA-F]{12}$'
```
[the remark this answers is not in the document]
Comment (kjduensing) [reply app/swagger/swagger/requests/claim_letters.rb:45]: Ok. I noticed that that was true for the response, but is that also true for the request? Like Someone will type in `/v0/claim_letters/{7B9FE306BA-28EF-4012-875A-1B4EEBACE931}`?

[the remark this answers is not in the document]
Comment (jerekshoe) [reply app/swagger/swagger/requests/claim_letters.rb:45]: Currently yes. Whether it should be that way is a different discussion, but that is how it works now

[the remark this answers is not in the document]
Comment (jerekshoe) [reply app/swagger/swagger/requests/claim_letters.rb]: Btw, we anticipate that the braces will come in as %7B ({) and %7D (}) and not as { or }. I'm not sure if that matters, but just wanted to point that out

**Pull request merged**